### PR TITLE
Added useful info when using WARP on a tightly-firewalled corporate network

### PR DESCRIPTION
**Pull request by jamie-sandbox**
Added useful info learned from hours of debugging and reverse-engineering when setting up WARP on a tightly-firewalled corporate network.

### CODE REVIEW

**Comment by ranbel** [comment]
@dh-cf

**Comment by jamie-sandbox** [comment]
Any progress on this?
________________________________

**Comment by ranbel** [comment]
This PR is awaiting a technical review from the WARP team.

**Comment by ranbel** [comment]
@aw-cf Could someone on your team help review this PR?

File: content/cloudflare-one/connections/connect-devices/warp/deployment/firewall.md [hunk 1]
```diff
@@ -74,7 +80,12 @@ If your organization does not currently allow inbound/outbound communication ove
 - macOS: `/Applications/Cloudflare WARP.app/Contents/Resources/CloudflareWARP`
 
 ### Optional scopes
-To run [Digital Experience Monitoring tests](/cloudflare-one/insights/dex/tests/), you will also need to allow the `warp-dex` process to generate network traffic to your target destinations:
+To run [Digital Experience Monitoring tests](/cloudflare-one/insights/dex/tests/), you will need to allow the `warp-dex` process to generate network traffic to your target destinations:
 
 - Windows: `C:\Program Files\Cloudflare\Cloudflare WARP\warp-dex.exe`
 - macOS: `/Applications/Cloudflare WARP.app/Contents/Resources/warp-dex`
+
+In order to allow the network connectivity tests within the WARP GUI to function reliably, you will also need to allow the userspace GUI to generate network traffic:
```
Comment (kkrum) [inline content/cloudflare-one/connections/connect-devices/warp/deployment/firewall.md:88]: This is a good addition we should keep

File: content/cloudflare-one/connections/connect-devices/warp/deployment/firewall.md [hunk 2]
```diff
@@ -56,9 +62,9 @@ The following domains are used as part of our captive portal check:
 
 ## Connectivity check
 
-As part of establishing the WARP connection, the client will check the following URLs to validate a successful connection:
+As part of establishing the WARP connection, the client will check the following HTTPS URLs to validate a successful connection:
 
-- `engage.cloudflareclient.com` verifies general Internet connectivity outside of the WARP tunnel.
+- `engage.cloudflareclient.com` verifies general Internet connectivity outside of the WARP tunnel. These requests are always sent directly and will not use a proxy server, even if one is configured for the system. Resolves to `162.159.192.1` and `2606:4700:d0::a29f:c001`.
```
Comment (kkrum) [inline content/cloudflare-one/connections/connect-devices/warp/deployment/firewall.md:67]: Pending our QA team checking re the proxy comment. We aren't doing anything special here so need to investigate if this is specific to certain configurations or if this is just how Windows decides to handle things.

We should document that the ips engage.cloudflareclient.com use though are whatever WARP Ingress IP we connect to. It isn't always what is in this PR, but could be anything in the WARP Ingress IPv4/24 or IPv6/48 range

File: content/cloudflare-one/connections/connect-devices/warp/deployment/firewall.md [hunk 3]
```diff
@@ -10,16 +10,22 @@ If your organization uses a firewall or other policies to restrict or intercept
...
 
+Once the tunnel has been established, communication with the API will take place **inside the tunnel**, unless a proxy configuration (e.g. PAC file) says otherwise.
+
 ## DoH IP
 
 All DNS requests through WARP are sent outside the tunnel via DoH (DNS over HTTPS). The following IP addresses must be reachable for DNS to work correctly.
 
 {{<render file="warp/_doh-ips.md">}}
 
+DoH requests are always sent directly and will not use a proxy server, even if one is configured for the system.
```
Comment (kkrum) [inline content/cloudflare-one/connections/connect-devices/warp/deployment/firewall.md:27]: Lets leave this out for now

File: content/cloudflare-one/connections/connect-devices/warp/deployment/firewall.md [hunk 4]
```diff
@@ -10,16 +10,22 @@ If your organization uses a firewall or other policies to restrict or intercept
...
 ## Client orchestration API
 
-The WARP client talks with our edge via a standard HTTPS connection outside the tunnel for operations like registration or settings changes. To perform these operations, you must allow `zero-trust-client.cloudflareclient.com` which will lookup the following IP addresses:
+Prior to connection, the WARP client talks with our edge via a standard HTTPS connection **outside the tunnel** for operations like registration and settings changes.
+
+Connections to the API will honour the system proxy settings (if configured), however it is recommended that the connections bypass any proxy and are allowed to be made directly. In this case you must allow `zero-trust-client.cloudflareclient.com` which will resolve to the following IP addresses:
 
 {{<render file="warp/_client-orchestration-ips.md">}}
 
+Once the tunnel has been established, communication with the API will take place **inside the tunnel**, unless a proxy configuration (e.g. PAC file) says otherwise.
```
Comment (kkrum) [inline content/cloudflare-one/connections/connect-devices/warp/deployment/firewall.md:19]: This is inaccurate. Orchestration API and DoH traffic is *always* outside the tunnel.

File: content/cloudflare-one/connections/connect-devices/warp/deployment/firewall.md [hunk 5]
```diff
@@ -10,16 +10,22 @@ If your organization uses a firewall or other policies to restrict or intercept
 
 ## Client orchestration API
 
-The WARP client talks with our edge via a standard HTTPS connection outside the tunnel for operations like registration or settings changes. To perform these operations, you must allow `zero-trust-client.cloudflareclient.com` which will lookup the following IP addresses:
+Prior to connection, the WARP client talks with our edge via a standard HTTPS connection **outside the tunnel** for operations like registration and settings changes.
```
Comment (kkrum) [inline content/cloudflare-one/connections/connect-devices/warp/deployment/firewall.md:13]: Lets hold off on this for now

File: content/cloudflare-one/connections/connect-devices/warp/deployment/firewall.md [hunk 6]
```diff
@@ -10,16 +10,22 @@ If your organization uses a firewall or other policies to restrict or intercept
...
+
+Connections to the API will honour the system proxy settings (if configured), however it is recommended that the connections bypass any proxy and are allowed to be made directly. In this case you must allow `zero-trust-client.cloudflareclient.com` which will resolve to the following IP addresses:
```
Comment (kkrum) [inline content/cloudflare-one/connections/connect-devices/warp/deployment/firewall.md:15]: Lets hold off on this for now

**Review by kkrum — CHANGES_REQUESTED** [review]
cc @ranbel added some comments based on discussion with engineering team.

**Comment by ranbel** [comment]
addressed in #14682

**Pull request closed without merging**